src/admin/content-block/components/ContentBlockForm/ContentBlockForm.tsx

```diff
@@ -93,7 +94,12 @@ const ContentBlockForm: FunctionComponent<ContentBlockFormProps> = ({
 			const errorArray = validator(updatedFormValue);
 
 			if (errorArray.length) {
-				errors[fieldKey] = errorArray;
+				if (isNumber(stateIndex)) {
+					errors[fieldKey] = [];
```

Comment:
Leeg je hier niet altijd de gehele errors array, die het probleem - die ik als comment op deze PR gaf - veroorzaakt?

Reply:
inderdaad

Meestal implementeer ik validatie als:
er is iets veranderd => valideer het hele formulier opnieuw

Want nu moet je de vorige state aanpassen en errors verwijderen uit een bestaand object. Dit lijkt me meer error prone dan gewoon het hele formulier opnieuw valideren. Maar mss ben ik nog teveel in de angular mindset :p